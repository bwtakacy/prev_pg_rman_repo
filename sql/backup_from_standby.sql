\! bash sql/backup_from_standby.sh

