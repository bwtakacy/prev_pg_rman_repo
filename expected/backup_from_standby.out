\! bash sql/backup_from_standby.sh
###### BACKUP COMMAND FROM STANDBY SERVER TEST-0001 ######
###### full backup mode ######
NOTICE:  pg_stop_backup complete, all required WAL segments have been archived
0
0
0

###### BACKUP COMMAND FROM STANDBY SERVER TEST-0002 ######
###### full + incremental backup mode ######
NOTICE:  pg_stop_backup complete, all required WAL segments have been archived
0
0
NOTICE:  pg_stop_backup complete, all required WAL segments have been archived
0
0
0
